indico/modules/events/papers/controllers/management.py

```diff
@@ -64,3 +71,38 @@ def _process_PUT(self):
     def _process_DELETE(self):
         set_reviewing_state(self.event_new, request.view_args['reviewing_type'], False)
         return jsonify_data(enabled=False, flash=False)
+
+
+class RHScheduleCFP(RHManagePapersBase):
+    """Schedule the call for papers"""
+
+    def _process(self):
+        form = PapersScheduleForm(obj=FormDefaults(**paper_reviewing_settings.get_all(self.event_new)),
+                                  event=self.event_new)
+        if form.validate_on_submit():
+            rescheduled = self.event_new.cfa.start_dt is not None
+            schedule_cfp(self.event_new, **form.data)
+            if rescheduled:
+                flash(_("Call for papers has been rescheduled"), 'success')
+            else:
+                flash(_("Call for papers has been scheduled"), 'success')
+            return jsonify_data(flash=False)
```

Comment:
See `RHSwitchReviewingType` after rebase what to return to re-render the page instead of reloading